graphs/fleurys_algorithm.py

```diff
@@ -0,0 +1,110 @@
+from collections import defaultdict
+
+
+class Graph:
+
+    def __init__(self, vertices):
+        self.v = vertices
+        self.graph = defaultdict(list)
+
+    def addEdge(self, u, v):
+        """
+        Function to add edges to the graph
+        """
+        self.graph[u].append(v)
+        self.graph[v].append(u)
+
+    def rmvEdge(self, u, v):
```

Comment:
```suggestion
    def rmv_edge(self, u, v):
```
Real variable names and type hints, please.